Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+		yum -y install epel-release && \
+		yum -y install python2-pip && \
+    yum clean all
+
+ENV LANG=en_US.UTF-8
+ENV LANGUAGE=en_US.UTF-8
+ENV LC_ALL=en_US.UTF-8
+ENV PYTHONUNBUFFERED=0
+
+RUN pip install obal
+RUN obal setup
+
+RUN mkdir /opt/packaging
```

Comment:
should this be `mkdir -p` to avoid problems if `/opt` shall ever vanish from the base?